style.css

```diff
@@ -0,0 +1,309 @@
+html {
+  height: 800px;
+}
+
+body {
+  margin: 0;
+  height: inherit;
+  font-family: "Montserrat", sans-serif;
+  background: linear-gradient(
+    45deg,
+    rgba(8, 15, 26, 0.59),
+    rgba(17, 17, 46, 0.46)
+  );
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.wrapper {
+  height: inherit;
+  max-width: 1440px;
+  margin: auto;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.justify-content-between {
+  justify-content: space-between;
+}
+
+.col-6 {
+  flex: 0 0 auto;
+  width: 50%;
+}
+
+.btn {
+  border: 1px solid #fff;
+  border-radius: 5px;
+  width: 40px;
+  height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  color: white;
+  background: rgba(174, 181, 185, 0.5);
+  cursor: pointer;
+  outline: none;
+}
+
+.info {
+  height: 100px;
+}
+
+.map {
+  height: 100%;
+}
+
+.control-section {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  font-family: "Montserrat", sans-serif;
+}
+
+.control-panel {
+  padding: 10px 20px;
+}
+
+.control-btn {
+  border: 1px solid #fff;
+  border-radius: 5px;
+  width: 40px;
+  height: 40px;
+  font-weight: 700;
+  font-size: 14px;
+  text-align: center;
+  color: white;
+  background: rgba(174, 181, 185, 0.5);
+  cursor: pointer;
+}
+
+.refresh-btn {
+  background-image: url(./assets/img/refresh-bg.svg);
+}
+
+.language-btn {
+  margin: 0 10px;
+  padding-left: 10px;
+  width: 70px;
+}
+
+.language-select {
+  font-size: 14px;
+  color: white;
+  background: none;
+  border: none;
+}
+
+.fahrenheit-btn {
+  border-radius: 5px 0 0 5px;
+}
+
+.celsius-btn {
+  border-radius: 0 5px 5px 0;
+  margin-left: -5px;
+}
+
+.search {
+  display: flex;
+}
+
+.search-input {
+  width: 200px;
+  color: white;
+  border: 1px solid #fff;
+  border-right-width: 0;
+  background-color: rgba(76, 82, 85, 0.4);
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+  outline: none;
+}
+
+.active-btn {
+  background-color: rgba(76, 82, 85, 0.4);
+}
+
+.search-input::placeholder {
+  color: white;
+  font-weight: 400;
+  font-family: "Montserrat", sans-serif;
+}
+
+.voice-search {
+  width: 44px;
+  height: 44px;
+  font-size: 14px;
+  color: rgba(255, 255, 255, 0.4);
+  background-color: rgba(76, 82, 85, 0.4);
+  border: 1px solid #fff;
+  border-width: 1px 0;
+  outline: none;
+  cursor: pointer;
+}
+
+.search-btn {
+  background-color: rgba(174, 181, 185, 0.5);
+  border-left-width: 0;
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+  width: 101px;
+  height: 44px;
+  font-size: 14px;
+  cursor: pointer;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 700;
+}
+
+.map {
+  padding-top: 220px;
+  padding-right: 90px;
+}
+
+#map {
```

Comment:
It's not a good practice to have class and id with the same name. So, please, rename one of them